hack/make-release-artifacts.sh

```diff
@@ -102,6 +116,9 @@ main() {
     istio_manifests_file="${OUT_DIR}/istio-manifests.yaml"
     mk_istio_manifests > "${istio_manifests_file}"
     log "Written ${istio_manifests_file}"
+
+    mk_kustomize_base
+    log "Written Kustomize base"
```

Comment:
I know this "Written Kustomize base" log message sounds awkward, but I wanted to be consistent with the existing log messages.